tests/sentry/utils/sdk_crashes/test_event_stripper.py

```diff
@@ -161,6 +161,40 @@ def test_strip_event_data_keeps_exception_mechanism(self):
             },
         }
 
+    def test_set_in_app_only_for_sdk_frames(self):
+        frames = get_frames("SentryCrashMonitor_CPPException.cpp", sentry_frame_in_app=False)
+
+        system_frame_in_app = [
+            {
+                "abs_path": "/System/Library/PrivateFrameworks/UIKitCore.framework/UIKitCore",
+                "in_app": True,
+            }
+        ]
+
+        event_data = get_crash_event_with_frames(system_frame_in_app + list(frames))
+
+        event = self.create_event(
+            data=event_data,
+            project_id=self.project.id,
+        )
+
+        stripped_event_data = strip_event_data(event.data, CocoaSDKCrashDetector())
+
+        stripped_frames = get_path(
+            stripped_event_data, "exception", "values", -1, "stacktrace", "frames"
+        )
+
+        for stripped_frame in stripped_frames[0::-1]:
+            assert stripped_frame["in_app"] is False
+
+        cocoa_sdk_frame = stripped_frames[-1]
+        assert cocoa_sdk_frame == {
+            "function": "SentryCrashMonitor_CPPException.cpp",
+            "package": "/private/var/containers/Bundle/Application/59E988EF-46DB-4C75-8E08-10C27DC3E90E/iOS-Swift.app/Frameworks/Sentry.framework/Sentry",
```

Comment:
What does the UUID(?) after `/Application/` refer to? May this UUID become a flaky test?

Reply:
Why do you think the UUID could lead to a flaky test, @iker-barriocanal?

Reply:
All good in that case, I didn't know where the UUID was coming from.

Reply:
Thanks for asking 🙏, @iker-barriocanal.